README.md

```diff
@@ -1,30 +1,33 @@
 # EDJX C++ SDK
 
-This repository contains builds of the EDJX C++ SDK. The `libedjx.a` library
+This repository contains EDJX C++ SDK builds.
+See the [release page](https://github.com/edjx/edjx-cpp-sdk/releases)
+for all downloads. Builds are provided for use with specific
+[WASI SDK](https://github.com/WebAssembly/wasi-sdk) versions.
+
+The `libedjx.a` library
 in `lib/` and header files in `include/edjx/` are required to build EDJX C++
 serverless functions. Please see
-[EDJX C++ SDK example](https://github.com/edjx/edjfunction-example-cpp)
+[EDJX C++ SDK examples](https://github.com/edjx/edjsamples-cpp)
 on how to build and use EDJX C++ Serverless functions.
 
-## Version
+## Installation Instructions
 
-This EDJX C++ SDK build was built against WASI SDK version 12
-(see [wasi-sdk-12 release page](https://github.com/WebAssembly/wasi-sdk/releases/tag/wasi-sdk-12)).
+To install the EDJX C++ SDK system-wide into the `/opt` directory so that
+it can be used to build serverless applications, run:
 
```

Comment:
@dasebek Hi David, How about adding a line.. which says " EDJX C++ SDK can be directly downloaded from <<Release>> link and need to be copied to /opt directory.

Reply:
@venugopv I added a paragraph
```
The EDJX C++ SDK can be directly downloaded from the
[release page](https://github.com/edjx/edjx-cpp-sdk/releases).
Each build of the EDJX C++ SDK targets a specific WASI SDK version.
For example, `wasi-12` releases of the EDJX C++ SDK should be used with
[WASI SDK 12](https://github.com/WebAssembly/wasi-sdk/releases/tag/wasi-sdk-12).
```

It is up to the user where they need to store the downloaded SDK in their projects. We provide commands to download it to /opt. If they want to build our example applications (https://github.com/edjx/edjsamples-cpp), the README in that repo tells them to download the SDK to /opt or to update the Makefile.